### ISSUE TITLE
Error parsing message 

### ISSUE THREAD

**Issue by wwofq0** [ThScEo/AutoForge#10]
Hello!
When I want to login with the new release (0.3.4), and I get the following error message:

Error parsing message : Sajnáljuk, belső hiba történt! Próbálkozz újra később Sajnáljuk, belső hiba történt! Próbálkozz újra később

Sajnáljuk, belső hiba történt! Same in english: Sorry, there was an internal error! 

Error when login. Check your credentials or sid and user key

![Képernyőkép 2022-09-02 100145](https://user-images.githubusercontent.com/63362133/188088437-1807151d-892c-4e5e-a0bc-0f05cdfffa3e.png)

**Comment by Gerchan** [comment]
Definitely wrong credentials.

**Comment by wwofq0** [comment]
I follow the description exactly and all the values are same.

**Comment by ThScEo** [comment]
- I made some changes. You can download the release 0.3.5.
- I've tried on your server and it seems to be OK. Let me know. 

**If you encounter this issue again, please check carefully your SID and User Key, as explained in the description.**

**Comment by wwofq0** [comment]
Thanks for the new release and your response. So what I know:
- With my old acc. won't work any way - Get this error msg: Connection failed. Error when login. Check your credentials or sid and user key. 
- Created a new acc. on the HU6 server, and it works! I tried it with 2 worlds and it works.
- Created a new acc. on the ENx server, and it works!

I dont know, something wrong with my old acc (!?) I registered around 2019.
